documentation/API.md

```diff
@@ -124,6 +81,92 @@ Basic login endpoint that will also return (on success):
 }
 ```
 
+#### Get preferences endpoint:
+
+GET: `/get-prefs`
+
+Provides current preferences and language options for the current installation. Called by front-end at the very start of loading in order to configure the environment according to system settings.
+
+#### Get language endpoint:
+
+GET: `/language/<langauge-code>`
+
+Gets the set of localised strings for the core application (i.e. not customisable entities like templates).
+
+#### Verification endpoint
+
+GET: `/verify?uid=<uniqueid>`
+
+Sets the verification record at `<uniqueid>` to `true` and triggers configured Action. Used for verification via email, etc. See [Create Verification Action](List-of-Action-plugins.md) for more info.
+
+#### File download endpoint:
+
+GET: `/file?id=<uniqueId>`
+
+Usage: `GET` request with file database id as a URL query parameter.
+
+This is a public endpoint, but files all have a long uniqueId which should prevent unauthorized access.s
```

Comment:
```suggestion
This is a public endpoint, but files all have a long uniqueId which should prevent unauthorized access.
```

Reply:
Tried using this endpoint to get the file existing in `core_templates` and got this error:
![Screen Shot 2021-10-06 at 3 25 02 PM](https://user-images.githubusercontent.com/16461988/136130961-35545308-91f7-4d18-a881-48c055788daf.png)

 In the server was showing: 
relation "permissions_all" does not exist
Cannot parse JWT
Cannot parse JWT

path: `http://localhost:8080/api/public/file?id=CylhAzxRhSX_QjtArq3bi`